### PR TITLE
[WIP][Skylab]: upgrade code to symfony 3 and pimple 3

### PR DESCRIPTION
**Pull request by sandergo90**
Skylab uses old Pimple container and old Cilex version. To be able to add more functionality to skylab, we need to upgrade it to symfony 3 and Pimple 3. This PR will update everything.

### CODE REVIEW

File: src/Kunstmaan/Skylab/Skeleton/WebserverSkeleton.php [hunk 1]
```diff
@@ -91,7 +91,7 @@ public function postMaintenance()
                 $this->processProvider->executeSudoCommand("ln -sf " . $this->app["config"]["nginx"]["sitesavailable"] . "/" . $config->getFilename() . " " . $this->app["config"]["nginx"]["sitesenabled"] . "/" . $config->getFilename());
             }
         } else {
-            $this->writeToFirstHostFile();
+            //$this->writeToFirstHostFile();
```
Comment (benoitgeerinck) [inline src/Kunstmaan/Skylab/Skeleton/WebserverSkeleton.php]: Why did you remove this line?

**Comment by benoitgeerinck** [comment]
@sandergo90: there seems to be dependency with php 7.1 ... see tests in travis.

Could you have look please?

File: src/Kunstmaan/Skylab/Skeleton/WebserverSkeleton.php (hunk 1 above)
Comment (sandergo90) [reply src/Kunstmaan/Skylab/Skeleton/WebserverSkeleton.php]: I've readded it ;)
